src/main/java/net/dv8tion/jda/client/managers/GroupManagerUpdatable.java

```diff
@@ -0,0 +1,179 @@
+/*
+ *     Copyright 2015-2017 Austin Keener & Michael Ritter
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package net.dv8tion.jda.client.managers;
+
+import org.apache.http.util.Args;
+import org.json.JSONObject;
+
+import net.dv8tion.jda.client.entities.Group;
+import net.dv8tion.jda.client.entities.impl.GroupImpl;
+import net.dv8tion.jda.client.managers.fields.GroupField;
+import net.dv8tion.jda.core.JDA;
+import net.dv8tion.jda.core.entities.Icon;
+import net.dv8tion.jda.core.requests.Request;
+import net.dv8tion.jda.core.requests.Response;
+import net.dv8tion.jda.core.requests.RestAction;
+import net.dv8tion.jda.core.requests.Route;
+
+public class GroupManagerUpdatable {
+	
+	protected final GroupImpl group;
+	protected GroupField<String> name;
+	protected GroupField<Icon> icon;
+
+	public GroupManagerUpdatable(GroupImpl group)
+	{
+		this.group = group;
+		setupFields();
+	}
+	
+	/**
+     * The {@link net.dv8tion.jda.core.JDA JDA} instance of this Manager
+     *
+     * @return the corresponding JDA instance
+     */
+	public JDA getJDA() //why...?
```

Comment:
I guess a getJDA method in every class can be kind of useful sometimes now that I think of it...